### DISCUSSION TITLE
Future of this project?

### DISCUSSION THREAD

**Discussion by Nkmol** [google/zx#255] [General] [answered]
How much can we expect a long term ecosystem support? What is expected roadmap of this project? Will testing be a major team in the future? Is this also used internally by Google teams, will promotion of ZX be a prio?

Basically I am missing some grip for myself if we want to move our services to ZX.

**Comment by antonmedv** [comment]
This is not an official Google product. It is my personal project. How good do I support my projects? Take a look at my other project at https://medv.io

For example, https://deployer.org was created 8 years ago and still I still support it. 

You can considering supporting me on GitHub as well to motivate me even more.

**Reply by GRX** [reply]
It's really good to see that you're maintaining your repo's so well for so long.
But I can't think anything other than that the repo location is it bit misleading (calling it "Google ZX") where it isn't even a (official) product of Google itself.


Other than that, it's hard to determine the future because even if you do maintain it as well as you do, you could be struck by lightning tomorrow and then this repository will become unmanaged.

**Reply by antonmedv** [reply]
Well, as this repo in google org it will be just easy to some other engineer at google take ownership. 

As for lighting, plenty of other OSS what I built also potentially at risk) but github has feature for it too)

**Comment by TonyGravagno** [comment]
I accept the provided answer here, but I also see a lot of un-merged PRs that are gathering dust.

The code is actively maintained but the last release was 7 months ago.

I completely understand the challenges of FOSS, full-time employment, etc. I'm just concerned that the project hasn't yet reached a level of activity that makes using it a comfortable decision.

Can authorization for updates, and processing of PRs, be shared with other interested Google developers to improve confidence for adoption?

Thanks

**Reply by antonmedv** [reply]
Actually, v8 is now actively implemented by the plan in #589 by @antongolub

**Reply by antongolub** [reply]
Let's admit the fact: zx has become part of the technological landscape of many large companies, this factor is definitely positive for the project's perspective. We have both the resources and expertise to continue our work in the foreseeable future.

**Reply by TonyGravagno** [reply]
@antongolub Sounds good to me. If that's accurate, I propose posting that confirmation as a separate answer so that this Discussion question can be marked as answered by @Nkmol.

**Comment by antongolub** [comment, marked as answer]
Liveness metrics seem fine:
* https://github.com/google/zx/releases
* https://github.com/google/zx/commits/main/
